packages/kit/src/views/Market/MarketDetailV2/components/TokenActivityOverview/utils/createTimeRangeOption.ts

```
@@ -15,11 +15,24 @@ export function createTimeRangeOption(
   const priceChangePercent = tokenDetail?.[priceChangeKey];
   if (priceChangePercent) {
     const percentage = parseFloat(priceChangePercent);
+    const formattedValue = parseFloat(percentage.toFixed(2));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)